### ISSUE TITLE
Accessing the data

### ISSUE THREAD

**Issue by iftachg** [xw-hu/DAF-Net#2]
Hi,

Do you intend to make the data that you have used in the paper public?

**Comment by xw-hu** [comment]
Yes. We will make the data publicly available in the next few days. Please check our website. Thanks!

**Comment by xw-hu** [comment]
Our RainCityscapes dataset is available for download at the Cityscapes website: https://www.cityscapes-dataset.com/downloads/

**Comment by ShijianXu** [comment]
Hi, the website said that the dataset is available upon request. However, I sent an email to contact them, but no response.

Are there any other ways to download your dataset?

Thanks a lot.

**Comment by xw-hu** [comment]
Please send the email to "mail@cityscapes-dataset.net", since they have the copyright of the Cityscapes dataset.

**Comment by xw-hu** [comment]
Please send the email to "mail@cityscapes-dataset.net", since they have the copyright of the Cityscapes dataset.

**Comment by Ir1d** [comment]
Hi @xw-hu , there's no RainCityscapes in the list of https://www.cityscapes-dataset.com/downloads/ at this moment. Is there any other way to access the data according to the paper? Or for example can you share the code to generate the data? Thx

**Comment by xw-hu** [comment]
@lr1d, please carefully find the dataset on that website. 
Please also send the email to "mail@cityscapes-dataset.net" for the request to download this dataset, since they have the copyright of the Cityscapes dataset.

**Comment by Ir1d** [comment]
I see, it is at the bottom of the page, named leftImg8bit_trainval_rain.zip
Do you have plans to release the codes to generate the data? @xw-hu

**Comment by kuijiang94** [comment]
Good works. I have sent the request to "mail@cityscapes-dataset.net", but get no response yet. As time is pressing on the response, is there any other way to access the data according to the paper?

**Comment by xw-hu** [comment]
Since they have the copyright of the Cityscapes dataset, you can only find the dataset from their website.

**Comment by jacktkk** [comment]
> Good works. I have sent the request to "[mail@cityscapes-dataset.net](mailto:mail@cityscapes-dataset.net)", but get no response yet. As time is pressing on the response, is there any other way to access the data according to the paper?

Hi, bro. Have you received an email reply? @kuihua
